City-of-Helsinki/drupal-helfi-platform-config
# Added Remote video paragraph.

How to test:
- `composer require drupal/helfi_platform_config:dev-UHF-X_remote_video_paragraph && composer update drupal/hdbt`
- `make new` ( if the site is not up and running )
- `drush features:import helfi_content -y` // Revert the Helfi content configuration feature.
- Clear the caches 
- Check that you can add Remote video paragraph
